app/assets/stylesheets/admin/pages/aide/_faq.scss

```
@@ -51,6 +51,9 @@
       opacity: 1;
       z-index: 1;
     }
+    a {
```

On peux ajouter du style pour les liens visités, non ?
```suggestion
    a, a:visited {
```

---

Ah oui, bien vu ! J'ai corrigé